tests/modsys/nested/T15.icry.stdout

```diff
@@ -1,15 +1,6 @@
 Loading module Cryptol
 Loading module Cryptol
 Loading module T15
-[warning] at T15.cry:5:13--5:14
```

Comment:
I'm not sure how I feel about losing these warnings.  What do you think?

Reply:
I wondered about that too.  More generally, I wonder if we should issue a warning when a local name shadows an imported name.  This is what the comment on line 166 of `Renamer.hs` is about.

Reply:
I feel like it probably makes sense to do that, controlled by the `warnShadowing` option.